src/generator.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const generateRandomNumber = () => {
+  let generatedNumber = '';
```

Comment:
For future reference, it would be better to use a `new Set()` here instead of a string (''). You can't add duplicates to set and `set.has(number)` is more efficient than `string.includes(number)`. 

`.includes` searches through every symbol of string one by one, while set's `.has` checks directly.